### ISSUE TITLE
[Bug]: “Add all cars” feature  still add Owens Mclaren to the garage

### ISSUE THREAD

**Issue by Lemon2911** [ForzaMods/AIO#212]
### What version of the aio were you running?

2.0.5.8

### Game

FH4

### Game Version

Microsoft store

### Have i checked if the bug was reported?

Yes

### What is the bug?

As @ItsLogic replied to issue #48 in Feb 6, 2022, Owens mclaren should be added to the add all cars blacklist. However, using this feature will still add Owens mclaren to the Garage, which will result in a ban.

**Comment by ItsLogic** [comment]
Yeah there seems to be a new issue in FH4 where the car blacklist which previously worked no longer does and we don't yet know why.
It should be fixed eventually :tm:
